### DISCUSSION TITLE
Remove CLA language from IP Policy (for now)

### DISCUSSION THREAD

**Discussion by ebullient** [commonhaus/foundation#77] [Consensus Building]
### Proposal

Reviewing the IP Policy (again), and...

It's one thing to require a DCO. That's pretty typical these days.

CLAs are more complicated. Example language from IP Policy for OpenJSF:

> The technical governing body of each Project may provide for additional requirements with respect to contributions. For example, a Project may require that new code contributions to a Project be accompanied by an OpenJS Foundation CLA, available at https://openjsf.org/cla.

For CF to be able to deal with CLAs, I think we'd have to define one (for the foundation). See the following:

- https://openjsf.org/cla
- https://www.apache.org/licenses/contributor-agreements.html
- https://www.eclipse.org/legal/ECA.php

Do we want to start here? Is DCO enough?

At the moment, I'm favoring punting the CLA language, and we can revisit/revise that aspect of the policy when warranted.

### Voting group

@commonhaus/founders

### Concensus mechanism

Please use emoji reactions to indicate your support:
- **👍 (:+1:)** if it looks good to you
- **👀 (:eyes:)** if you're "ok" with it (it may not be your favorite)
- **👎 (:-1:)** if you think this needs discussion first

Ask questions or raise concerns in comments.

**Comment by kenfinnigan** [comment]
This wouldn't prevent projects from utilizing their existing CLA, correct?

**Reply by ebullient** [reply]
If we remove the language for now, that's correct. 

I think the question is what (or which) CLA we would use officially as a foundation going forward. Project-level CLA (by/for a project, like Netty's) would be good. A CLA for a specific company won't work as well.

Maybe we just adopt/adapt Netty's CLA for projects that want to use a CLA (in addition to a DCO)? The Foundation can then help track/manage who has signed what (corporate CLA vs. individual CLA, etc)

**Reply by kenfinnigan** [reply]
This approach makes sense, I'm good with punting on CLAs at foundation level for day-one

**Comment by ebullient** [comment]
Thanks for the input, Ken.

I'm moving this here, so we can be concrete about it:
https://github.com/commonhaus/foundation-draft/pull/78
